src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
+* Once you complete this step and create new user records, we highly recommend completing the migration process without delay. If you don't complete the steps to migrate assets and delete the original user record, a user may see two user records when logging in (see image below) or else may be missing assets they expect to see (like dashboards). 
+
+ ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
 
     <figcaption>
-If your email address is associated with multiple logins, you can verify your email to see those accounts. In the example above, the “Original New Relic account” refers to that user record being on the older, original account/user model. 
+If a user with access to multiple organizations is not migrated for all organizations, they'll retain an "original" user record, in addition to the new record created. When logging in, they'll have a choice of which to log into. 
     </figcaption>
+
+
+## Step 6: Access settings [#access-settings]
+
+![User migration page 6](./images/user-migration-page-6.png "user-migration-page-6.png")
+
+This step allows you to determine the level of access your users have. If you need to map user access to specific accounts and roles, then you'll want to set up groups and access grants at this stage. 
+
+You'll need to create an access grant for each account that you want a group to have access to. For example, a DevOps group might have five different access grants, each giving the group access to a different account.  
+
+This procedure is done separately from the user migration wizard procedure. For more on how to create access grants, see the [user management tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).
+
+## Step 7: Migrate user assets [#migrate-assets]
+
+![User migration page 7](./images/user-migration-page-7.png "user-migration-page-7.png")
+
+Upon completion of this step, personal assets, such as dashboards and favorites, are migrated to new user records and the original user record is deleted.
+
+If a user has access to several organizations (for example, if that user is a contractor), the user won't be fully deleted until all organizations migrate their users. It's possible that such a user will find themselves with both the original user record and one or more new user records (see image below). 
+
+ ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
```

Comment:
I don't think we need this image twice. I think it's fine to just have it the first time toward the start of this doc